src/main/java/com/javacountries/javacountries/Country.java

```diff
@@ -0,0 +1,67 @@
+package com.javacountries.javacountries;
+
+public class Country {
+    private String name;
+    private int population;
+    private int landAreaKm;
+    private int medianAge;
+
+    public Country(String name, int population, int landAreaKm, int medianAge) {
+        this.name =
+                name;
+        this.population =
+                population;
+        this.landAreaKm =
+                landAreaKm;
+        this.medianAge = medianAge;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name =
+                name;
+    }
+
+    public int getPopulation() {
+        return population;
+    }
+
+    public void setPopulation(int population) {
+        this.population =
+                population;
+    }
+
+    public int getLandAreaKm() {
+        return landAreaKm;
+    }
+
+    public void setLandAreaKm(int landAreaKm) {
+        this.landAreaKm =
+                landAreaKm;
+    }
+
+    public int getMedianAge() {
+        return medianAge;
+    }
+
+    public void setMedianAge(int medianAge) {
+        this.medianAge =
+                medianAge;
+    }
+
+    @Override
+    public String toString() {
+        return "Country{" +
+                "name='" +
```

Comment:
Did you intend to use this method at some point? If you did intend to use it when printing a collection, you'll need to create a new collection holding `String` types and then use a loop to assign each `Country` item with the `.toString` method used on it.

Of course, since you have a class as well for a `CountryList`, you could override the `.toString` method for it too.